### ISSUE TITLE
esp-01 на p25q80h. ошибки в работе

### ISSUE THREAD

**Issue by Banzaizaizai** [tretyakovsa/Sonoff_WiFi_switch#115]
Выглядит так: 
прошивается. ТД появляется,  вебинтерфейс поднимает. 
выбираем свою ТД, перезагружаем модуль,  на пару секунд плата регистрируется в сети и после этого отваливается

в эфире появляется ТД формата ESP-XXXXXX

если к ней подключится - доступен вебинтерфейс, но все пункты в нем - Undefined

скрин добавил
на хабре писали с этой микросхемой есть проблема с ФС
![2018-12-11](https://user-images.githubusercontent.com/13425455/49791285-d480b200-fd38-11e8-847a-a4a30f18ad68.png)

**Comment by Banzaizaizai** [comment]
https://habr.com/post/409911/
статья с хабра

**Comment by renat2985** [comment]
Да, вам не повезло с Flash Chip. Как заставить правильно писать в файловую систему PUYA чипы, я не понимаю.
Делал подобные эксперименты https://github.com/esp8266/Arduino/issues/4061#issuecomment-422849071 не помогает.

Если есть какие-то другие идеи пишите.
У меня тоже много ESP PUYA.

**Comment by renat2985** [comment]
@Shoorup4eg конкретно с этим проектом (Sonoff_WiFi_switch
) почему-то не работает.

**Comment by renat2985** [comment]
@Banzaizaizai, попробуйте эти bin файлы на ESP с PUYA Flash чипом.
Только прошивайте в режиме DOUT. И отпишитесь как работают.

build http://backup.privet.lv/esp/Sonoff_WiFi_switch.ino.generic.puya.bin
spiffs http://backup.privet.lv/esp/Sonoff_WiFi_switch.spiffs.bin

**Comment by Banzaizaizai** [comment]
в понедельник возьму из магазина, прошью и отпишусь что да как.. самому интересно. пока что поменялся на человеческую версию

**Comment by Banzaizaizai** [comment]
попробовал. не поменялась ситуация. регистрация в сети на пару секунд и досвидос.  поднялась ТД ESPxxxx

**Comment by renat2985** [comment]
@Banzaizaizai у меня аналогично. Ладно, может что-то еще в голову со временем придет.

**Comment by antkachenko** [comment]
У меня тоже таких модулей много пришло. Решаю проблему покупкой модулей памяти Winbond в Chip и Deep и перепайкой...
